### ISSUE TITLE
Change label for automatic jdk upgrade issues

### ISSUE THREAD

**Issue by mbhave** [spring-projects/spring-boot#15825]
It uses a special label at the moment and we probably don't need that. It could either be `type: task` or `waiting-for-triage` since there is no milestone assigned.

**Comment by mbhave** [comment]
We've decided to add the waiting-for-triage and task labels.

**Comment by mbhave** [comment]
I just remembered why I created a separate label for this. We don't want to create an issue if there's an issue that already exists for it. We check [weekly for updates](https://ci.spring.io/teams/spring-boot/pipelines/spring-boot/jobs/detect-jdk-updates/builds/8), so there's a chance we would end up with the same issue multiple times.

If anyone has other ideas for tracking these issues other than a custom label, we can try that.
